src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -1,4 +1,19 @@
 package com.spartronics4915.frc2020;
 
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.I2C.Port;
+
 public final class Constants {
-}
+    public static final int kPanelRotatorBeamSensorUpID = -1;
```

Comment:
I know I just told you to write them this way, but.. our new convention is to use a _subclass_  for each subsystem, which has the upside of making `PanelRotator` not needed in the variable names. [See here for an example.](https://github.com/Spartronics4915/2020-InfiniteRecharge/blob/master/src/main/java/com/spartronics4915/frc2020/Constants.java)

It'll look like this:
```
public static final class PanelRotator {
    public static final int kBeamSensorUpId = -1;
    public static final int kBeamSensorDownId = -1;
    etc...
```

Sorry!

Reply:
Done! I'll push in a minute.